nemo/collections/llm/quantization/quantizer.py

```diff
@@ -51,13 +54,6 @@
         "int4": mtq.INT4_BLOCKWISE_WEIGHT_ONLY_CFG,
     }
 
-    HAVE_MODELOPT = True
-
-except (ImportError, ModuleNotFoundError) as e:
-    HAVE_MODELOPT = False
-    HAVE_MODELOPT_ERROR = e
-
-
 SUPPORTED_DTYPE = [16, "16", "bf16"]  # Default precision for non-quantized layers
```

Comment:
If it's an Enum, let's define it as an enum in the future. No updated needed right now. Adding it just for future references.

Reply:
OK. I'll take a note on this for now